docs/develop/api-reference/collection-ttl.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 4
+sidebar_label: CollectionTTL object
+title: CollectionTTL object API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/collectionttl
+---
+
+# CollectionTTL object API reference
+
+This object is passed in to API methods to say whether a TTL should be updated and if so, what the new TTL value should be.
+
+This tries to make the process a little less bewildering by providing named constructors and copiers for various situations.
+
+See [Expire Data with TTL](../../../learn/how-it-works/expire-data-with-ttl), For more information on how TTL works with Momento Serverless Cache, 
+
+## Common method behaviors
+
+- If a CollectionTTL is not passed, CollectionTtl.fromCacheTtl(), from the cache connection object, is used.
+
+## Constructor parameters
+
+- ttl: duration - (optional) if no TTL is given, the TTL set in the client connection object is used
+- refreshTtl: boolean = true - if true, the collection’s TTL will be reset to the ttl
```

Comment:
may be worth also specifying the behavior for the false case?  "Otherwise, the collection's existing TTL will be retained"